json-spec/examples/landsat8-sample.json

```diff
@@ -35,92 +35,195 @@
     },
     "B4": {
-      "href": "LC81530252014153LGN00_B4.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-4.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B4.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["4"]
     },
     "B5": {
-      "href": "LC81530252014153LGN00_B5.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-5.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B5.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["5"]
     },
     "B6": {
-      "href": "LC81530252014153LGN00_B6.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-6.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B6.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["6"]
     },
     "B7": {
-      "href": "LC81530252014153LGN00_B7.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-7.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B7.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["7"]
     },
     "B8": {
-      "href": "LC81530252014153LGN00_B8.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-8.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B8.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["8"]
     },
     "B9": {
-      "href": "LC81530252014153LGN00_B9.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-9.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B9.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["9"]
     },
     "B10": {
-      "href": "LC81530252014153LGN00_B10.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-10.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B10.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["10"]
     },
     "B11": {
-      "href": "LC81530252014153LGN00_B11.TIF",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product-band-11.json"
+      "href": "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B11.TIF",
+      "type": "GeoTIFF",
+      "required": true,
+      "eo:bands": ["11"]
     },
-    "metadata": {
-      "href": "LC81530252014153LGN00_MTL.txt",
-      "product" : "http://landsat-pds.s3.amazonaws.com/L8/landsat8_product_metadata.json"
+
+    "eo:bands": {
```

Comment:
This should also be moved to Collection, right?

Reply:
Oh, yeah this whole example incorporates the whole collection level metadata, which I think is okay to illustrate the complete record. But then we should also have an example with an Item and then a Collection record separated.